core/icon.js

```diff
@@ -143,9 +143,11 @@ Blockly.Icon.prototype.updateColour = function() {
 /**
  * Render the icon.
  * @param {number} cursorX Horizontal offset at which to position the icon.
+ * @param {?number} cursorY Vertical offset at which to position the icon.
  * @return {number} Horizontal offset for next item to draw.
  */
-Blockly.Icon.prototype.renderIcon = function(cursorX) {
+Blockly.Icon.prototype.renderIcon = function(cursorX, cursorY) {
```

Comment:
Let's refactor these changes to `renderIcon()` into their own method so that we don't need to repeat the unrelated steps

Reply:
Sure.